src/utils/url.ts

```diff
@@ -0,0 +1,112 @@
+import { Request } from "express";
```

Comment:
Do we use anything in this file anywhere? If not why do we need it?

Reply:
This is not used at the moment for this WI, will take it out for now.